### PR TITLE
feat: add configureSession & userId prop

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.atlassian.net/browse/AC-252

### What's the request? 
Open an interface that allows users to pass their own custom session handler. 

### How I handled it?
I've added new props, `configureSession` and `userId`, which can be passed into the `<SendbirdProvider />` component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,11 +186,90 @@ const App = () => {
...
 export default App;
 
 ```
+
+## Advanced Customization
+```jsx
+import { ChatAiWidget } from "@sendbird/chat-ai-widget";
+import '@sendbird/chat-ai-widget/dist/style.css';
+
+import { SessionHandler } from '@sendbird/chat'
```
Comment (AhyoungRyu) [inline README.md:202]: @sravan-s which one do you recommend? 
```
import SessionHandler from '@sendbird/uikit-react/handlers/SessionHandler'
```
or 
```
import { SessionHandler } from '@sendbird/chat'
```

Comment (sravan-s) [reply README.md:202]: import { SessionHandler } from '@sendbird/chat' > this  is recommended


---  
#1  was a work around for an issue we found in v3 release(I will share ticket)

Comment (sravan-s) [reply README.md:202]: https://sendbird.atlassian.net/browse/CORE-2964

**Review by sravan-s — APPROVED** [review]

**Review by luke-cha — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (AhyoungRyu) [reply README.md:202]: Ah I see. Thanks!

**Pull request merged**